rand_core/src/block.rs

```diff
@@ -54,7 +57,8 @@ use crate::impls::{fill_via_u32_chunks, fill_via_u64_chunks};
 use crate::{CryptoRng, Error, RngCore, SeedableRng};
 use core::convert::AsRef;
 use core::fmt;
-#[cfg(feature = "serde1")] use serde::{Deserialize, Serialize};
+#[cfg(feature = "serde1")]
+use serde::{Deserialize, Serialize};
```

Comment:
Did we change our minds about this? IIRC, you rejected such a change in an older PR.

Reply:
The formatting? I don't care so much, and consider it more *art* (find something nice or at least not terrible) than science.

My stance *has* changed slightly. With KAS I now use `rustfmt` (mostly stock config) and sometimes adjust code (e.g. by introducing extra `let` bindings) to achieve a nice result.